corporate/lib/stripe.py

```diff
@@ -3140,11 +3146,12 @@ def is_sponsored(self) -> bool:
 
     @override
     def get_metadata_for_stripe_update_card(self) -> Dict[str, Any]:
-        return {
+        metadata = {
             "type": "card_update",
-            # TODO: Add user identity metadata from the remote realm identity
-            # "user_id": user.id,
         }
+        if self.remote_billing_user is not None:
+            metadata["user_id"] = str(self.remote_billing_user.id)
+        return metadata
```

Comment:
This can also assert that we have one -- it'll always be the case when logged in.